src/handlers/serenity.rs

```diff
@@ -128,18 +128,18 @@ impl SerenityHandler {
                         .description("Add a track to the queue")
                         .default_permission(true)
```

Comment:
Not just playlists, I think that if the rule of thumb is we should lock the command if it modifies the queue, then lock it because in a big server not everyone should spam songs.

In #154, Rafael suggested an approach that I really like. Please take a look, you wouldn't need to check for role. 🙂

Reply:
In fact, on #148 both João and Rafael agree it should be restricted so if that's also okay with you, let's lock it.